Answers.md

```diff
@@ -1,11 +1,14 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
-
+    ReactJS is a Javascript library primarily used for creating interactive UI interfaces and front-end aspects of an application. React is an attempt to solve problems related to the costs of DOM manipulation. Through the use of a virtual DOM a developer can mitigate these costs. Specifically, through the use of react's architecture only components that have had state changes need be re-rendered.
 1.  What does it mean to _think_ in react?
+    Thinking in React is a process of contemplating how you can break down an app into individual components while making determinations about state and how you wish to manipulate your components.
 
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
+    Class components handle data as it navigates the React application. Components are responsible for defining and organizing the data. Functional components in contrast are used to display chosen data through functions and commands that dictate presentation to an end user.
 
 1.  Describe state.
-
+    State refers to the values of the moving parts on an app when rendered, in other words, the data being changed on a given component.
 1.  Describe props.
+    Props refers to properties which is data that is passed around through the use of components.

```

Comment:
Probs is the state that you take from the parent component that you would pass down to the child components. 👍